sdk/flutter/quickstart.mdx

```diff
@@ -0,0 +1,157 @@
+---
+title: Turso Quickstart (Flutter / Dart)
+sidebarTitle: Quickstart
+description: Get started with Turso and Flutter / Dart using the libSQL client in a few simple steps
+---
+
+In this Flutter/Dart quickstart we will learn how to:
+
+- Retrieve database credentials
+- Install the Flutter/Dart libSQL client
+- Connect to a local or remote Turso database
+- Execute a query using SQL
+- Sync changes to local database (optional)
+
+<Steps>
+
+  <Step title="Retrieve database credentials">
+
+You will need an existing database to continue. If you don't have one, [create one](/quickstart).
+
+<Snippet file="retrieve-database-credentials.mdx" />
+<Info>You will want to store these as environment variables.</Info>
+
+  </Step>
+
+  <Step title="Install libsql_dart">
+
+```bash
+flutter pub add libsql_dart
+```
+
+  </Step>
+
+  <Step title="Initialize">
+
+Now connect to your local or remote database using the `LibsqlClient` constructor:
+
+<AccordionGroup>
+  <Accordion title="Embedded Replicas">
+    ```dart
+    final dir = await getApplicationCacheDirectory();
+    final path = '${dir.path}/local.db';
+
+    final client = LibsqlClient(path)
+      ..authToken = '<TOKEN>'
+      ..syncUrl = '<TURSO_OR_LIBSQL_URL>'
+      ..syncIntervalSeconds = 5
+      ..readYourWrites = true;
+    ```
+
+  </Accordion>
+
+<Accordion title="Local only">
+
+  <CodeGroup>
+      ```dart File
+      final dir = await getApplicationCacheDirectory();
+      final path = '${dir.path}/local.db';
+
+      final client = LibsqlClient(path);
+      ```
+
+      ```dart In Memory
```

Comment:
this should be in a separate group?

Reply:
@dikatok I was thinking of updating other SDK Quickstarts to show the In-Memory variant inside the Local only story. So I think it's fine to group them here, wdyt?